cmsis/CMSIS_5/CMSIS/RTOS2/RTX/Source/TOOLCHAIN_ARM/TARGET_RTOS_M4_M7/irq_cm4f.S

```diff
@@ -28,7 +28,7 @@
 RTX_STACK_CHECK EQU      0
 #endif
 
-                IF       ({FPU}="FPv4-SP")
+                IF       ({FPU}="FPv4-SP") || ({FPU}="VFPv4_SP_D16") || ({FPU}="FPv5-SP") || ({FPU}="FPv5_D16")
```

Comment:
"This will be fixed upstream in the next version of CMSIS." Could you provide a link as you did with other "fixed in next version" patches?

Reply:
added tracking issue to the commit msg